Bottom: 131551fb9b47bfc7b909d72692fcc8dfb91ea942
Top:    6140dfe94e12f1d13290506b3e7725ba5901ded8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-13 01:55:44 +0100

Refresh of add-support-for-arc-segments

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 572a138..b9d654c 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2510,7 +2510,6 @@ poly_IniContour (PLINE * c)
     return;
   /* bzero (c, sizeof(PLINE)); */
   c->head.next = c->head.prev = &c->head;
-#warning BROKEN ASSUMPTION of 32 BIT
   c->xmin = c->ymin = 0x7fffffff;
   c->xmax = c->ymax = 0x80000000;
   c->is_round = FALSE;
@@ -2904,7 +2903,6 @@ poly_InsideContour (PLINE * c, Vector p)
   info.f = 0;
   info.p[0] = ray.X1 = p[0];
   info.p[1] = ray.Y1 = p[1];
-#warning BROKEN ASSUMPTION of 32 BIT
   ray.X2 = 0x7fffffff;
   ray.Y2 = p[1] + 1;
   if (setjmp (info.env) == 0)
